package.json

```diff
@@ -35,11 +35,11 @@
   "dependencies": {
     "regenerator-runtime": "^0.13.9",
     "shp-write": "^0.3.2",
-    "vue": "^2.6.12",
+    "vue": "2.6.12",
     "vue-cookie-law": "^1.13.3"
   },
-  "resolutions": {
-    "graceful-fs": "4.2.3"
+  "overrides": {
+    "graceful-fs": "^4.2.10"
```

Comment:
Please also add some notes about this change in the description of the pull request